{{ cookiecutter.package_name }}/docs/conf.py

```diff
@@ -69,15 +70,14 @@
 # This does not *have* to match the package name, but typically does
 project = setup_cfg['name']
 author = setup_cfg['author']
-copyright = '{0}, {1}'.format(
-    datetime.datetime.now().year, setup_cfg['author'])
+copyright = '{0}, {1}'.format(datetime.datetime.now().year, author)
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 
-import_module(setup_cfg['name'])
-package = sys.modules[setup_cfg['name']]
+import_module(project)
```

Comment:
shouldn't it be the other way around, the importable is the module, and project should be the generic name? With a quick look there seem to be some inconsistencies remained, but I haven't double checked it yet. It's certainly confusing.

Reply:
I don't know. I am just doing a simple variable replacement here, since `project` is already set on L71. I think @astrofrog said he is going to clean up `conf.py` at some point.

Reply:
yes, it's definitely not a comment for this PR.

Reply:
Yeah I'll clean this up in a follow-up PR :)